# This represents a list of ellipsoids
string[] names
Ellipsoid[] ellipsoids